core/src/fr/hammons/slinc/Tools.scala

```diff
@@ -27,16 +27,20 @@ object Tools:
 
   def sendResourceToCache(name: String): Unit =
     Files.createDirectories(appDataStore)
-    val cacheLocation = appDataStore.resolve(s"$name.c")
-
+    val cachedSharedLibLocation = appDataStore.resolve(s"$name$sharedLibSuffix")
```

Comment:
I want to pick up a shared library for the platform where program is running and send it to `appDataStore.resolve(s"$name$sharedLibSuffix")`.